File: Belajar_Day_1/day1_6.py
while True :
    lanjut = input("Apakah lanjut? (y/n): ").lower()
    if lanjut in ['y']: # contoh klo lower
        print("Halo")
        continue
    elif lanjut == 'n' or lanjut =='N': # contoh if biasa
        break
    else :
        print("input tidak sesuai")
        break

File: Belajar_Day_1/day1_6.java
import java.util.Scanner;

public class day1_6 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        boolean lanjut = true;
        String halo = "Halo";
        while (lanjut){
            System.out.println(halo); 
            System.out.println("Apakah lanjut? (y/n)");
            String input = scan.nextLine();

            if (input.equalsIgnoreCase("y")){
                continue;
            } else if (input.equalsIgnoreCase("n")){
                break;
            } else {
                System.out.println("Input tidak sesuai");
            }
        }
        scan.close();
    }
}
